Dockerfile

```diff
@@ -227,9 +279,9 @@ WORKDIR /build
 COPY . .
 
 COPY --from=builder /build/sources ./sources/
-COPY --from=grpc /build/grpc ./grpc/
+COPY --from=grpc /opt/grpc /usr/local
```

Comment:
It looks good to me, I just can't see where /opt/grpc is used then? as we don't run `make install` anymore here, where it is being picked up?

Reply:
The make install is being run in the grpc container now

Reply:
thanks!